src/Core/View.js

```diff
@@ -512,4 +513,82 @@ View.prototype.normalizedToViewCoords = function normalizedToViewCoords(ndcCoord
     return _eventCoords;
 };
 
+/**
+ * Return objects from some layers/objects3d under the mouse in this view.
```

Comment:
returns only object3D?

Reply:
See the @return documentation for the returned values.